### DISCUSSION TITLE
AWS Deployment - Wiki keeps redirecting to installation page

### DISCUSSION THREAD

**Discussion by charles-crawford** [requarks/wiki#6702] [Error / Bug Report] [answered]
I am installing on AWS with Aurora Serverless v1 and ECS Fargate.  I keep getting redirected to the installation page after a "successful installation" message even tho "login" is at the end of my url. I can eventually get to the login page if I just keep trying the login url. I can create pages, users, groups, etc. but I will randomly get sent back to the installation page as well. 

Logs from service:

Not Found TypeError: Cannot read properties of undefined (reading 'sendError') at /wiki/server/setup.js:394:20

September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:09.016Z [MASTER] [32minfo[39m: Optional extension sharp was not found on this system. [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
-- | -- | -- | --
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:09.015Z [MASTER] [32minfo[39m: Optional extension puppeteer was not found on this system. [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:09.009Z [MASTER] [32minfo[39m: Optional extension pandoc was not found on this system. [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:09.003Z [MASTER] [32minfo[39m: Optional extension git is installed. [ OK ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:08.991Z [MASTER] [32minfo[39m: No new storage targets found: [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:08.992Z [MASTER] [32minfo[39m: Checking for installed optional extensions... | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:08.949Z [MASTER] [32minfo[39m: No new search engines found: [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:08.914Z [MASTER] [32minfo[39m: No new renderers found: [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:08.818Z [MASTER] [32minfo[39m: No new loggers found: [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js
September 06, 2023, 20:15 (UTC-6:00) | 2023-09-07T02:15:08.773Z [MASTER] [32minfo[39m: No new editors found: [ SKIPPED ] | ce1276cfe0464939bb730a6582307df8 | wiki-js

DB set up:
Engine: aurora-postgresql, EngineMode: serverless, EngineVersion: 13.9. I have all security groups opened up to debug this. 

Please let me know if you need any ore information regarding my deployment. Thanks

**Comment by NGPixel** [comment]
Sounds like you're spinning up new instances that connects to a new empty DB. Make sure it always connect to the same DB.

**Reply by charles-crawford** [reply]
@NGPixel Thanks for the quick response. I have everything in a VPC and am creating just one DB with 1 count for the task/container. Do you have any other thoughts? Thanks

**Comment by charles-crawford** [comment, marked as answer]
Reason for closing: 

On a second look I had a DesiredCount: 2 for my Fargate Service. So it seems what is happening is that there are files being written locally to the container(s). I'm guessing that using EFS and redirecting those writes to EFS will solve the problem I had. Then multiple containers could be deployed for failover and/or high availability. For now I just dropped the DesiredCount to 1.
